### PR TITLE
Update DG, add all relevant use cases.

### PR DESCRIPTION
**Pull request by Ty-stan0417**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -446,171 +446,351 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-    * *a1. CLM clears the command line using the backspace key.<br>
+* *a. At any time, library manager chooses to cancel to list all users.<br>
+    * *a1. Library manager clears the command line using the backspace key.<br>
 
   ***Use case ends.***
 
+
 #### Use case: UC3 - Edit user's information
+#### MSS:
+1. Library manager requests to edit a user's information.
```
Comment (jinhanfromNUS) [inline docs/DeveloperGuide.md:497]: I feel "Library manager **requests** to edit a user's information." should change to "Library manager **intends** to edit a user's information."

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -446,171 +446,351 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 
+* *a. At any time, library manager chooses to cancel the edition of user.<br>
+    * *a1. Library manager clears the command line using the backspace key.<br>
+
+  ***Use case ends.***
+
+
+#### Use case: UC4 - Find users
 #### MSS:
+1. Library manager requests to find users by keyword.
```
Comment (jinhanfromNUS) [inline docs/DeveloperGuide.md:528]: I feel **requests** can be changed to **intends**.

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -446,171 +446,351 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+* *a. At any time, library manager chooses to cancel the donate process of user.<br>
+    * *a1. Library manager clears the command line using the backspace key.<br>
+
+  ***Use case ends.***
+
+
+#### Use case: UC9 - Add a book to the library
+#### MSS:
+1. Library manager requests to add a book to the library.
```
Comment (jinhanfromNUS) [inline docs/DeveloperGuide.md:664]: I feel **requests** can be changed to **intends**.

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -446,171 +446,351 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+    * *a1. Library manager clears the command line using the backspace key.<br>
 
   ***Use case ends.***
 
-**Use case: Borrower borrows a book from the library**
 
-**MSS**
+#### Use case: UC10 - Delete a book from the library
+#### MSS:
+1. Library manager requests to delete a book from the library.
```
Comment (jinhanfromNUS) [inline docs/DeveloperGuide.md:688]: I feel **requests** can be changed to **intends**.

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -446,171 +446,351 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 
-  Use case ends.
+  ***Use case ends.***
 
-* 4a. The given index is invalid.
 
-    * 4a1. MyBookshelf shows an error message.
+#### Use case: UC11 - View the limit for libary
+#### MSS:
+1. Library manager requests to view the limit for libary.
```
Comment (jinhanfromNUS) [inline docs/DeveloperGuide.md:712]: I feel **requests** can be changed to **intends**.

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -446,171 +446,351 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-**MSS**
 
-1.  Borrower requests to return a book.
-2.  Librarian requests to list users.
-3.  MyBookshelf shows the list of all users.
-4.  Librarian removes the book from the borrower's borrow book list.
-5.  MyBookshelf updates the borrower's details.
+#### Use case: UC12 - Set the limit for libary
+#### MSS:
+1. Library manager requests to set the limit for libary.
```
Comment (jinhanfromNUS) [inline docs/DeveloperGuide.md:727]: I feel **requests** can be changed to **intends**.

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -446,171 +446,351 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+#### Extensions:
+* *a. At any time, library manager chooses to cancel help.<br>
+    * *a1. Library manager clears the command line using the backspace key.<br>
+
+  ***Use case ends.***
+
+
+#### Use case: UC15 - Exit MyBookshelf
+#### MSS:
+1. Library manager requests to exit.
```
Comment (jinhanfromNUS) [inline docs/DeveloperGuide.md:781]: I feel **requests** can be changed to **intends**.

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (Ty-stan0417) [reply docs/DeveloperGuide.md]: Changed in new commit.

**Comment by Ty-stan0417** [comment]
Changed some of requests -> intends
in new commit.

**Comment by jinhanfromNUS** [comment]
> Changed some of requests -> intends in new commit.

I just realized that user should change to library user as well. No need to worry about the old ones (I have already created a pull request for that). Just need to change the user to library user in current commit.

**Review by jinhanfromNUS — APPROVED** [review]

**Pull request merged**